app/src/components/pages/collectionsPage/collectionsPage.tsx

```diff
@@ -37,20 +146,111 @@ export const CollectionsPage = ({ data }) => {
           subtitle="Explore the New York Public Library's diverse collections, including digitized photographs, manuscripts, maps, and more. Start exploring by using the search bar below or browse through the collections."
         />
         <SearchBar
-          sx={{ maxWidth: "462px", marginTop: "l" }}
+          sx={{ maxWidth: "462px" }}
```

Comment:
Where did you get this number from?

Reply:
https://github.com/NYPL/digital-collections/pull/180/files#diff-db9636018921e1dbb9ce8c307175c049f72c22d0add5d992ef77ef35a2ca68c1R38 

![Screenshot 2024-12-10 at 1 51 22 PM (2)](https://github.com/user-attachments/assets/8f4d8704-3505-4d8c-882c-a9b90fe6c582)

Reply:
Yes, that's fair. You may need to do 
```
sx={{
  "> input": { maxWidth: "462px" }
}}
```
This is setting the max width to the entire component: 
<img width="512" alt="Screenshot 2024-12-11 at 9 13 19 AM" src="https://github.com/user-attachments/assets/a00a4fc2-974e-4e20-b60c-7dbfc98640bc" />